ImageFeed/Auth/AuthViewController.swift

```diff
@@ -0,0 +1,34 @@
+import UIKit
+
+protocol AuthViewControllerDelegate: AnyObject{
+    func authViewController(_ vc: AuthViewController, didAuthenticateWithCode code: String)
+}
+
+final class AuthViewController: UIViewController{
+    let WebViewSegueIdentifier = "ShowWebView"
+    
+    weak var delegate: AuthViewControllerDelegate?
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == WebViewSegueIdentifier{
```

Comment:
у фигурных скобочек должны быть оступы, не должен код сливаться в одну сплошную лапшу:
```suggestion
        if segue.identifier == WebViewSegueIdentifier {
```

Reply:
Исправил